webapp/blog/logic.py

```diff
@@ -108,7 +108,7 @@ def change_url(feed, host):
     :returns: A string with converted urls
     """
     url_regex = re.compile(
-        "https:\/\/admin.insights.ubuntu.com(\/\d{4}\/\d{2}\/\d{2})?"
+        "https:\\/\\/admin.insights.ubuntu.com(\\/\\d{4}\\/\\d{2}\\/\\d{2})?"
```

Comment:
Why not use raw strings?

```suggestion
        "https:\/\/admin.insights.ubuntu.com(\/\d{4}\/\d{2}\/\d{2})?"
```

Also, are you absolutely sure slashes even need escaping? I don't see why they would.
``` bash
robin@xps:~/git/juju-docs (master)$ ipython3 
impPython 3.6.6 (default, Sep 12 2018, 18:26:19) 
Type 'copyright', 'credits' or 'license' for more information
IPython 6.5.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import re

In [2]: re.match("/hello/", "/hello/world")
Out[2]: <_sre.SRE_Match object; span=(0, 7), match='/hello/'>
```